tasks/steps/blocks/update_package.yml

```diff
@@ -0,0 +1,54 @@
+---
+
+- name: 'Get package meta info'
+  any_errors_fatal: true
+  cartridge_set_package_info:
+    package_path: '{{ delivered_package_path }}'
+    app_name: '{{ cartridge_app_name }}'
+
+- name: 'Set "cartridge_app_name" from package info'
+  set_fact:
+    cartridge_app_name: '{{ package_info.name }}'
+
+- name: 'BLOCK : Enable Tarantool repository'
```

Comment:
It wouldn't be displayed, yes?

Reply:
Yes, because it's a block, not include task

Reply:
But some `BLOCK` names are displayed. Let's remove names for Ansible blocks.

Reply:
Names are displayed for include tasks